proto/pocket/service/relay.proto

```diff
@@ -37,10 +37,13 @@ message RelayRequest {
 // JSONRPCRequestPayload contains the payload for a JSON-RPC request.
 // See https://www.jsonrpc.org/specification#request_object for more details.
 message JSONRPCRequestPayload {
-    bytes id = 1; // Identifier established by the Client to create context for the request.
+    uint32 id = 1; // Identifier established by the Client to create context for the request.
     string jsonrpc = 2; // Version of JSON-RPC. Must be exactly "2.0".
     string method = 3; // Method being invoked on the server.
-    map<string, string> parameters = 4; // Parameters for the method. https://www.jsonrpc.org/specification#parameter_structures
+    // TODO_RESEARCH: Find out why the params being a map causes errors
```

Comment:
#126 Look into using a `oneof` here regarding parameters in the JSON-RPC request